models/room.go

```diff
@@ -15,5 +15,7 @@ type EmergencyRoomBase struct {
 type EmergencyRoom struct {
 	ID uuid.UUID
 	EmergencyRoomBase
-	Departments []Department `gorm:"many2many:rooms_have_departments" json:"-"`
+	OrganizationID uuid.UUID
+	Organization   Organization
```

Comment:
Shouldn't there be a GORM annotation here? Take a look at https://github.com/helpwave/rest-api/blob/main/models/emergency.go#L14.

Reply:
Nice, thanks for the clarification 👍